LBH.AdultSocialCare.Api/V1/Gateways/Payments/Concrete/PayRunInvoiceGateway.cs

```diff
@@ -89,6 +90,43 @@ public PayRunInvoiceGateway(DatabaseContext dbContext)
             return await BuildPayRunInvoiceQuery(query, fields).SingleOrDefaultAsync();
         }
 
+        public async Task<decimal> GetPayRunInvoicedTotalAsync(Guid payRunId)
```

Comment:
No need to go DB for each metric. 

Let's get PayrunInvoices with related tables then calculate all metrics.